File: data/code_pairs/fortran/395.f90
! hello.f90
program hello
    call greet
    call performCalculation
end program hello

subroutine greet
    print *, "Hello, World!"
end subroutine greet

subroutine performCalculation
    ! This subroutine calculates the sum of 3 and 4 and prints the result.
    integer :: result
    result = 3 + 4
    print *, "The sum of 3 and 4 is", result, "."
end subroutine performCalculation

File: data/code_pairs/cpp/395.cpp
// hello.cpp
#include <iostream>

void greet();
void performCalculation();

int main() {
    greet();
    performCalculation();
    return 0;
}

void greet() {
    std::cout << "Hello, World!" << std::endl;
}

void performCalculation() {
    // This function calculates the sum of 3 and 4 and prints the result.
    int result = 3 + 4;
    std::cout << "The sum of 3 and 4 is " << result << "." << std::endl;
}